bin/kn-provision

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+echo "Inside kn-provision"
```

Comment:
Please change with something line "Provisioning bla bla...", IMO we shouldn't assume that the user knows the internal structure of the project